test/pensions/config/childrenInformation.unit.spec.jsx

```diff
@@ -73,7 +73,7 @@ describe('Child information page', () => {
     formDOM.submitForm(form);
     const errors = formDOM.querySelectorAll('.usa-input-error-label');
 
-    errors.forEach(e => console.log(e.getAttribute('for'))); // eslint-disable-line no-console
+    // errors.forEach(e => console.log(e.getAttribute('for'))); // eslint-disable-line no-console
```

Comment:
Dug into the mocha reporter and there was not an easy way to stop logs from interfering with the reporter without filtering the `stderr` or `stdout` streams. I divert the `stderr` stream while mocha is running- there were too many errors for me to fix easily right now. This was the only place where `stdout` was interfering with the reporter.

These logs don't feed into the reporter and failing tests have enough information on their own to debug most scenarios- we should probably remove this.

Reply:
That seems reasonable to me.